pkg/core/alias.go

```diff
@@ -15,6 +18,7 @@ var (
 	NewLogger = kuma_log.NewLogger
 	SetLogger = kube_log.SetLogger
 	Now       = time.Now
+	LookupIP  = dns.MakeCaching(net.LookupIP, 10*time.Second)
```

Comment:
please parametrize this 10 seconds. I have a feeling this will be useful value to change while debugging stuff

Reply:
do you mean to put this value into Config? Then I probably have to move LookupIP initialization into RuntimeBuilder or something like that, is it okay?

Reply:
Added new parameter - `Config.GeneralConfig.DNSCacheTTL`, not sure about place for this parameter in the config